apps/metadata-utils/src/types.ts

```diff
@@ -104,3 +104,10 @@ export type columnValue = string | number | boolean | columnValueObject;
 interface columnValueObject {
   [x: string]: columnValue;
 }
+
+export type IInputValue = string | number | boolean;
```

Comment:
I think the IInputValue should be integrated with columnValue types?
- primitiveColumValue = string | number | boolean;
- columnValue = primitiveColumnValue | columnValueObject | columnValueObject[]
- columnValueObject = [x: string]: columnValue